ci_framework/roles/openshift_setup/tasks/main.yml

```diff
@@ -61,4 +61,5 @@
         podman login
         -u {{ cifmw_openshift_user }}
         -p {{ cifmw_openshift_token }}
+        --tls-verify=false
```

Comment:
if *really*, please don't hard-code it, but add a new parameter. I'm still convince we shouldn't need it - so adding a parameter allows an easy toggle if it blocks anything in the CI.

Reply:
Done